src/components/biggive-form-field-select/biggive-form-field-select.scss

```diff
@@ -54,7 +54,47 @@ select.grey {
   text-align: left;
   &.select-style-bordered {
     .sleeve {
```

Comment:
the dropdown() mixin wasn't used anywhere else, so I inlined the code. I hope that means `.sleeve:focus-within` part I added at the end is a more understandable.